test/libsolidity/syntaxTests/functionCalls/base_overload.sol

```diff
@@ -0,0 +1,20 @@
+contract Base {
+	uint public x;
+	uint public y;
+	function init(uint a, uint b) public {
+		x = a;
+		y = b;
+	}
+	function init(uint a) public {
+		x = a;
+	}
+}
+
+contract Child is Base {
+	function init(uint c) public {
+		Base.init(c);
```

Comment:
This used to give an error (wrong argument count) because `init(uint a)` was ignored